frontend/src/utils/video.ts

```diff
@@ -124,59 +133,83 @@ export async function getVideoFromPreviousComparisons(
         offset: 0,
       })
     )?.count || 0;
+
   const comparisonVideoResult = await UsersService.usersMeComparisonsList({
     pollName: YOUTUBE_POLL_NAME,
     limit: 99,
     offset: Math.floor(Math.random() * comparisonCount),
   });
+
   const cl = comparisonVideoResult?.results || [];
   const comparisonVideoList = [
-    ...cl.map((v) => v.entity_a.metadata.video_id),
-    ...cl.map((v) => v.entity_b.metadata.video_id),
+    ...cl.map((v) => v.entity_a.uid),
+    ...cl.map((v) => v.entity_b.uid),
   ];
-  const comparisonVideoId = retryRandomPick(
-    5,
-    otherVideo,
-    currentVideo,
-    comparisonVideoList
+
+  let newSuggestions = comparisonVideoList.filter(
+    (uid, index) => comparisonVideoList.indexOf(uid) === index
   );
```

Comment:
Instead of deduplicating comparisons, maybe we can fetch the last compared entities using the ratings API?

Reply:
Fixed https://github.com/tournesol-app/tournesol/pull/1758/commits/dc43c042ab8b794d1cecc70547a3f8b0c0eb28e8